app/models/user.rb

```diff
@@ -1,6 +1,63 @@
 class User < ActiveRecord::Base
```

Comment:
Bunch of issues with this class. Have you heard of the gem `reek`? https://github.com/troessner/reek It's really useful for keeping your code clean.

Once you have it installed run : `reek app/models/user.rb`